tests/test_state_change_block.py

```diff
@@ -76,3 +76,44 @@ def test_state_name_property(self):
         blk.process_signals([StateSignal(1, 'A')])
         self.assertDictEqual(self.last_notified[DEFAULT_TERMINAL][0].to_dict(),
                              {'prev_attr': None, 'attr': 1, 'group': None})
+
+    def test_state_change_not_persist(self):
+        """Test that initial state is not persisted"""
+        blk = StateChange()
+        self.configure_block(blk, {
+                'name': 'test_block',
+                'state_expr': '{{$state}}',
+                'initial_state': '{{1}}',
+                'group_by': '{{$group}}',
+                'exclude': False
+            })
+
+        blk.start()
+        #get state, causing it to be saved
```

Comment:
Check out the other comments in these files. Python comments usually have a space directly after the `#`